backend/ethereum/channel/conclude.go

```diff
@@ -100,6 +95,25 @@ func (a *Adjudicator) ensureConcluded(ctx context.Context, req channel.Adjudicat
 	}
 }
 
+func (a *Adjudicator) conclude(ctx context.Context, req channel.AdjudicatorReq, subStates channel.StateMap) error {
+	// If the on-chain state resulted from forced execution, we do not have a fully-signed state and cannot call concludeFinal.
+	forceExecuted, err := a.isForceExecuted(ctx, req.Params.ID())
+	if err != nil {
+		return errors.WithMessage(err, "checking force execution")
+	}
+	if req.Tx.IsFinal && !forceExecuted {
+		err = errors.WithMessage(a.callConcludeFinal(ctx, req), "calling concludeFinal")
+	} else {
+		err = errors.WithMessage(a.callConclude(ctx, req, subStates), "calling conclude")
+	}
+	if IsErrTxFailed(err) {
+		a.log.Warn("Calling conclude(Final) failed, waiting for event anyways...")
+	} else if err != nil {
```

Comment:
I dont think you need this _else if_.

Reply:
we do, because we don't want to fail, if it's an `ErrTxFailed`.